OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -90,6 +90,7 @@ public class CrushClass { }
 	public interface ICrushable
 	{
 		bool CrushableBy(Actor self, Actor crusher, BitSet<CrushClass> crushClasses);
+		bool IsCrushable(Actor self, BitSet<CrushClass> crushClasses, out LongBitSet<PlayerBitMask> blocking);
```

Comment:
My confusion above could be resolved by changing this to return the `LongBitSet<PlayerBitMask>` instead, or renaming and commenting as needed to make it clear why and how !crushable is not equivalent to blocking.